awscognito/cognito_users.py

```diff
@@ -0,0 +1,171 @@
+# pylint: disable=global-statement,redefined-outer-name
+""" Script used to create|disable AWS Cognito user """
+import argparse
+import sys
+from dataclasses import dataclass
+
+import boto3
+import pandas
+import yaml
+
+
+@dataclass
+class User:
+    """ Class for AWS Cognito user """
+
+    # pylint: disable=too-many-instance-attributes
```

Comment:
You may not need this line if using python 3.7?

Reply:
Updated.